find_nearer_stores/find_nearer_stores.py

```diff
@@ -0,0 +1,331 @@
+# -*- coding: utf-8 -*-
+
+import math
+import os
+import pandas as pd
+import re
+import requests
+from urllib.parse import urlencode
+
+__author__ = 'Jingwen ZHENG'
+
+WORK_DIR = '/Users/jingwen/Documents/python/datasets/understaffing_overstaffing'
+STARTS_WITH_DIGIT = re.compile('^\\d.*')
+STARTS_WITH_A = re.compile('^A .*')
+
+
+def complete_zipcode(zipcode):
+    if len(zipcode) == 1:
+        zipcode = zipcode
+    else:
+        rep_time = 5 - len(zipcode)
+        zipcode = '%s%s' % ('0' * rep_time, zipcode)
```

Comment:
Concatenate 5 leading zeros to the zip-code, then return only the 5 digits on the right side:
 
```python
return ('00000' + zipcode)[-5:]
```